backend/data_validation/utils.py

```diff
@@ -7,10 +8,24 @@
 class ErrorType(Enum):
     """Enum class for the different error types that can occur with the data validation."""
 
-    RESCOUT_MATCH = 0
-    INCORRECT_DATA = 1
-    MISSING_DATA = 2
-    CRITICAL = 3
+    DEBUG = 0
+    INFO = 1
+    INCORRECT_DATA = 2
+    MISSING_DATA = 3
+    CRITICAL = 4
+    RESCOUT_MATCH = 5
+
+
+@dataclass
+class Constants:
+    @dataclass
+    class RapidReact:
+        AUTO_TAXI_POINT_VALUE = 2
+        AUTO_UPPER_HUB_POINT_VALUE = 4
+        AUTO_LOWER_HUB_POINT_VALUE = 2
+
+        TELEOP_UPPER_HUB_POINT_VALUE = 2
+        TELEOP_LOWER_HUB_POINT_VALUE = 1
```

Comment:
Perhaps we should move this to a constants file and not make nested classes?

Reply:
implemented